File: Destructor/cpp/Persona.cpp
#include "Persona.h"

Persona::Persona(string name, string last, int age){
    nombre = name;
    apellidos = last;
    edad = age;
}

Persona::~Persona(){
    cout<<"Me mori Dx"<<endl;
}

void Persona::caminar(){
    cout<<"Estoy caminando"<<endl;
}

void Persona::dormir(){
    cout<<"zzz"<<endl;
}

void Persona::hablar(){
    cout<<"Hola, soy "<<nombre<<endl;
}

File: Destructor/python/Persona.py
class Persona(object):
    def __init__(self,name,last,age):
        self.nombre = name
        self.apellido = last
        self.edad = age

    def hablar(self):
        print("Hola, soy {}".format(self.nombre))

    def caminar(self):
        print("Estoy caminando")

    def dormir(self):
        print("zzz")

    def __del__(self):
        print("Me mori Dx")